osu.Game/Screens/Backgrounds/BackgroundScreenDefault.cs

```diff
@@ -60,8 +119,24 @@ private Background createBackground()
         {
             Background newBackground;
 
-            if (user.Value?.IsSupporter ?? false)
-                newBackground = new SkinnedBackground(skin.Value, backgroundName);
+            if (User.Value?.IsSupporter ?? false)
```

Comment:
this is ugly to read and has code duplication. you're looking for something like this

![image](https://user-images.githubusercontent.com/191335/62837187-74ffeb00-bca7-11e9-987e-f980a6aa31b6.png)